python_modules/libraries/dagster-celery/dagster_celery/executor.py

```diff
@@ -92,3 +94,33 @@ def celery_enabled_pipeline():
         include=init_context.executor_config.get('include'),
         retries=Retries.from_config(init_context.executor_config['retries']),
     )
+
+
+def celery_docker_config():
+    additional_config = {
+        'repo_location_name': Field(
+            StringSource,
+            is_required=False,
+            default_value=IN_PROCESS_NAME,
```

Comment:
does this engine work with an in process repo? i would expect not...

Reply:
Why do we need this repo_location_name anyway? I found that it is equal to repo_name at most of the cases and could not make out when it differs and why it is necessary. Could you please explain what is it for?